PrinceOfVersionsMacSample/PrinceOfVersionsMacSample/Swift Implementation/ConfigurationController.swift

```diff
@@ -56,15 +56,20 @@ private extension ConfigurationController {
 
         let princeOfVersionsURL = URL(string: Constants.princeOfVersionsURL)!
 
-        PrinceOfVersions.checkForUpdates(from: princeOfVersionsURL, options: options, completion: { [weak self] response in
-            switch response.result {
-            case .success(let updateResultData):
-                self?.fillUI(with: updateResultData)
-            case .failure:
-                // Handle error
-                break
+        PrinceOfVersions.checkForUpdates(
+            from: princeOfVersionsURL,
+            options: options,
+            cachePolicy: .reloadIgnoringLocalAndRemoteCacheData,
```

Comment:
Since you already provided the default value in the function declaration, should we remove the explicit cache policy setting here? 

The same comment applies to all other call sites of this function.

Reply:
I guess we don't need to since it's mentioned in the documentation 😄 
I will remove it 👌